pkg/rpc/datacatalogservice/service.go

```diff
@@ -68,7 +68,7 @@ func (s *DataCatalogService) AddTag(ctx context.Context, request *catalog.AddTag
 func NewDataCatalogService() *DataCatalogService {
 	configProvider := runtime.NewConfigurationProvider()
 	dataCatalogConfig := configProvider.ApplicationConfiguration().GetDataCatalogConfig()
-	catalogScope := promutils.NewScope(dataCatalogConfig.MetricsScope).NewSubScope("service")
+	catalogScope := promutils.NewScope(dataCatalogConfig.MetricsScope).NewSubScope("datacatalog")
```

Comment:
so it'll say `flyte:datacatalog` ? not `datacatalog:something` ?

Reply:
Yes, the first level can be `flyte` if that's what you set in the `config.yaml`.